src/pages/_document.page.tsx

```diff
@@ -10,7 +10,7 @@ export default function Document() {
   return (
     <Html>
       <Head>
-        <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;600&family=Noto+Serif:wght@400;600" rel="stylesheet"/>
+        <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;600&family=Noto+Serif:wght@400;700" rel="stylesheet"/>
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
```

Comment:
This looks good to me.

Is there any reason why we would not use an include for the link fonts to avoid duplication?